Library/Homebrew/shims/super/cc

```diff
@@ -310,6 +311,9 @@ class Cmd
     args += path_flags("-isystem", isystem_paths) + path_flags("-I", include_paths)
     # Add -nostdinc when building against glibc@2.13 to avoid mixing system and brewed glibc headers.
     args << "-nostdinc" if @deps.include?("glibc@2.13")
+    # Ideally this would be -ffile-prefix-map, but that requires a minimum of GCC 8, LLVM Clang 10 or Apple Clang 12
+    # and detecting the version dynamcally based on what `HOMEBREW_CC` may have been rewritten to point to is awkward
+    args << "-fdebug-prefix-map=#{formula_buildpath}=." if formula_buildpath
```

Comment:
Any particular reason why we want `.` here?

Reply:
No real reason - this can be anything we want it to be. Though an empty string doesn't seem to work.

Debian uses `.`. If we want more info we could add the formula name or something.